File: src/Omnia.ProcessManagement.Models/ProcessTypes/ReviewReminderSchedule/PropertySchedule.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.ProcessTypes.ReviewReminderSchedule
{
    public class PropertySchedule : ReviewReminderSchedule
    {
        public override ReviewReminderScheduleTypes Type
        {
            get
            {
                return ReviewReminderScheduleTypes.Property;
            }
        }

        public Guid DateTimeEnterprisePropertyDefinitionId { get; set; }
    }
}

File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processtypes/reviewreminderschedule/PropertySchedule.ts
import { ReviewReminderSchedule, ReviewReminderScheduleTypes } from './ReviewReminderSchedule';
import { GuidValue } from '@omnia/fx-models';

export interface PropertySchedule extends ReviewReminderSchedule {
    type: ReviewReminderScheduleTypes.Property
    dateTimeEnterprisePropertyDefinitionId: GuidValue;
}